### PR TITLE
Add/promtail

### LINKED ISSUE #3: Add promtail to monitoring-role

### PR DESCRIPTION
**Pull request by raneq**
Closes #3 #4 #6 
~~Must rebase the branch from master.~~ done with _raneq force-pushed the add/promtail branch from 9d8da30 to 3ff0b25 now_

### CODE REVIEW

**Comment by raneq** [comment]
It's working :) 
imatge

It doesn't matter that the backups fail, the test is that logs get to grafana from monitoring role

File: templates/monitoring-docker-compose.yml.j2 [hunk 1]
```diff
@@ -9,8 +9,26 @@ services:
...
+    restart: always
+    ports:
+      - {{ monitoring_promtail_docker_bind}}
+    volumes:
+      - {{ monitoring_promtail_config_dir }}:{{ monitoring_promtail_config_dir }}
+{% for volume in monitoring_promtail_docker_volumes %}
+      - {{ volume }}
+{% endfor %}
+#    healthcheck:
+#      test: 'curl http://{{ monitoring_promtail_host }}:{{ monitoring_promtail_port }}/ready'
```
Comment (raneq) [inline templates/monitoring-docker-compose.yml.j2]: Can you think about a reason why this healthcheck was failing?
Node exporter returns a 200 if done manually, yet docker says unhealthy.

On the other hand, promtail is listening to the port, but returns empty replies, with no HTTP content (no headers, error code, etc), and then keeps restarting.

cc/ @sauloperez @danypr92

Comment (raneq) [reply templates/monitoring-docker-compose.yml.j2]: The `ready` endpoint is documented [here](https://github.com/grafana/loki/tree/master/docs/clients/promtail#api)

Comment (sauloperez) [reply templates/monitoring-docker-compose.yml.j2]: then something is really broken if it does not return HTTP things. Should we create an issue for it?

**Comment by raneq** [comment]
Add/promtail and a changelog file

**Comment by raneq** [comment]
Add/promtail and a changelog file

**Pull request merged**